px-k8s-auto-resize/6.fill-and-resize-volume.md

```diff
@@ -0,0 +1,49 @@
+In this step, we will run a benchmark that uses more than 20% of our volume and show how AutoPilot dynamically increases the volume size without downtime or user intervention.
+
+### Step: Open a shell inside the postgres container
+
+Below commands exec into the postgres pod:
+
+```
+POD=`kubectl get pods -l app=postgres | grep Running | grep 1/1 | awk '{print $1}'`
+kubectl exec -it $POD bash
+```{{execute T1}}
+
+Next we can launch the psql utility and create a database
+```
+psql
+create database pxdemo;
+\l
+\q
+```{{execute T1}}
+
+Use pgbench to run a baseline transaction benchmark which will try to grow the volume to a size that is greater than the 20% that we defined in our AutoPilot Rule. This should trigger AutoPilot to resize the volume.
+
+```
+pgbench -i -s 50 pxdemo
+```{{execute T1}}
+
+* Note that once the test completes, **AutoPilot will make sure the usage remains above 20% for about 30 seconds before triggering the rule.** Click ```exit```{{execute T1}} top exit from the pod shell before proceeding.
+
+### Step: Check to see if the rule was triggered
+
+We can retrieve events by using the `kubectl get events` and filtering for `AutoPilotRule` events that match our use case. Note, that AutoPilot delays the rule being triggered immediately, so make sure to **hang tight and see the rule get triggered if you dont see it right away**.
```

Comment:
can be renamed as 

`that AutoPilot delays the rule from being triggered immediately to ensure that the conditions stablize`